grpc/tests/src/test/deploy/receipts.rs

```diff
@@ -88,7 +89,7 @@ fn should_record_wasmless_transfer() {
 
     // TODO: this is borked
```

Comment:
How is this "borked"?

Reply:
That comment is mine, and likely can be considered vestigial.  If you run our system code (mint, auction, etc.) as contracts, it will have a different cost than if you run it as host-side code.  Our tests run in both modes, so we can't make this assertion in the host-side case.